pkgs/jovian-greeter/greeter.py

```diff
@@ -166,16 +178,19 @@ def _consume_session(self) -> Optional[str]:
         return next_session
 
     def _find_sessions(self, sessions: Iterable[str]) -> Optional[Session]:
-        for data_dir in self.xdg_data_dirs + [ '/usr/share' ]:
+        for data_dir in self.xdg_data_dirs + [ '/usr/share', '/run/current-system/sw/share' ]:
```

Comment:
/run/current-system/sw/share should generally always be on XDG_DATA_DIRS, and if it isn't, we should figure out why.

Reply:
Hmm. It seems in the path by the time I get to the shell or UI, so it's most likely that the environment `greeter.py` is running in has something wrong with it. Is the concern here that the path might change?

Reply:
The concern is that this is unnecessary, and in a year we won't remember why we put it there.